common/OpTestSystem.py

```diff
@@ -618,3 +624,132 @@ def sys_execute_cmd_onto_bmc(self, i_cmd):
             return self.cv_BMC.sys_execute_cmd_onto_bmc(i_cmd)
         except OpTestError as e:
             return BMC_CONST.FW_FAILED
+
+
+    ###########################################################################
+    # IPMI Console interfaces
+    ###########################################################################
+
+    ##
+    # @brief It will get the ipmi sol console
+    #
+    # @return l_con object if success or raise OpTestError
+    #
+    def sys_get_ipmi_console(self):
+        self.l_con = self.cv_IPMI.ipmi_get_console()
+        return self.l_con
+
+    ##
+    # @brief This function make sure, ipmi console is activated and then login to the lpar if host is already up.
+    #
+    # @param i_con @type string: this is the console object used for ipmi sol console access and for lpar login.
```

Comment:
I don't understand how this can be a string.

Please give an example in this case. Thanks